src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/Options/ToastOptions.shared.cs

```diff
@@ -18,6 +18,11 @@ public class ToastOptions
 
 		public static MessageOptions DefaultMessageOptions { get; set; } = new MessageOptions();
 
+		/// <summary>
+		/// Show the SnackBar above AnchorView.
+		/// </summary>
+		public View AnchorView { get; set; }
+
```

Comment:
```suggestion
#if __ANDROID__
		/// <summary>
		/// Show the SnackBar above AnchorView.
		/// </summary>
		public View AnchorView { get; set; }
#endif
```
Tried this but failed (in line AnchorView = Fab,) even that I was targeting Android, any idea why?

Reply:
Because that is shared code. And it doesn't target Android (it targets netstandard).
If you decided to implement extension method you shouldn't add this property at all.

Reply:
@VladislavAntonyuk  it is not important now because I don't need it, but got me confused when saw it being used in https://github.com/xamarin/XamarinCommunityToolkit/blob/b0ca201f766da86a3d53fdddeccd6ced79319592/src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Touch/TouchEffect.shared.cs#L470 it is shared code also, isn't ?